sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -36,16 +36,31 @@ import org.apache.spark.sql.types._
 /**
  * Re-run all the tests in SQLQueryTestSuite via Thrift Server.
  *
+ * Each case is loaded from a file in "spark/sql/core/src/test/resources/sql-tests/inputs".
+ * Each case has a golden result file in "spark/sql/core/src/test/resources/sql-tests/results".
+ *
  * To run the entire test suite:
  * {{{
- *   build/sbt "hive-thriftserver/testOnly *ThriftServerQueryTestSuite" -Phive-thriftserver
+ *   build/sbt -Phive-thriftserver "hive-thriftserver/testOnly *ThriftServerQueryTestSuite"
+ * }}}
+ *
+ * To run a single test file upon change:
+ * {{{
+ *   build/sbt -Phive-thriftserver \
+ *     "hive-thriftserver/testOnly *ThriftServerQueryTestSuite -- -z inline-table.sql"
+ *
  * }}}
  *
  * This test suite won't generate golden files. To re-generate golden files for entire suite, run:
  * {{{
  *   SPARK_GENERATE_GOLDEN_FILES=1 build/sbt "sql/testOnly *SQLQueryTestSuite"
  * }}}
  *
+ * To re-generate golden file for a single test, run:
```

Comment:
Such command can run multiple tests, for example:
```
SPARK_GENERATE_GOLDEN_FILES=1 build/sbt "sql/testOnly *SQLQueryTestSuite -- -z datetime.sql"
```
updates:
```
./sql/core/src/test/resources/sql-tests/inputs/ansi/datetime.sql
./sql/core/src/test/resources/sql-tests/inputs/datetime.sql
```

Reply:
Yeah I know. This is copied from `SQLQueryTestSuite`.
If the developer really wants to update the single test only, he/she has to use the exact test case name. Otherwise, "-z e.sql" can update multiple test outputs too.